pkg/nuctl/command/deploy.go

```diff
@@ -218,6 +219,7 @@ func newDeployCommandeer(ctx context.Context, rootCommandeer *RootCommandeer, be
 	}
 
 	addDeployFlags(cmd, commandeer)
+	cmd.Flags().BoolVarP(&commandeer.withImage, "with-image", "i", false, "Flag to leave image value in function config")
```

Comment:
Although it is true, it's a bit confusing in the `deploy` command.
Maybe we can change it to `skip-spec-cleanup` or something similar.

Reply:
@TomerShor renamed but only for project